refactor(curriculum): restructure sudoku solver python project (#54303)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/07-scientific-computing-with-python/learn-classes-and-objects-by-building-a-sudoku-solver/6606c2d203a8124c83b2234b.md

---
id: 6606c2d203a8124c83b2234b
title: Step 31
challengeType: 20
dashedName: step-31
---

# --description--

You need to check if a given number is not equal to the number in the specified column of the current row.

> For that, replace `pass` with a generator expression that iterates over the range from `0` to `8` (included) and for each `row` evaluates if the number at the row `row` and column `col` in the board is different from `num`.

Reviewer: ```suggestion
For this, replace `pass` with a generator expression that iterates over the range from `0` to `8` (inclusive), and for each `row`, evaluates whether the number at the specified `row` and column `col` on the board is different from `num`.
```
